src/test/kotlin/racingCar/domain/UsernameTest.kt

```diff
@@ -0,0 +1,12 @@
+package racingCar.domain
+
+import org.junit.jupiter.api.Assertions.assertThrowsExactly
+import org.junit.jupiter.api.Test
+
+class UsernameTest {
+
+    @Test
+    fun `사용자 이름은 빈값이 아닙니다`() {
+        assertThrowsExactly(IllegalArgumentException::class.java, { Username("") })
```

Comment:
다음과 같이 좀 더 간결하게 작성 가능하겠네요~
```suggestion
        assertThrows<IllegalArgumentException> { Username("") }
```
추가적으로 에러 메세지도 같이 테스트해주면 더 좋을 거 같아요!